client/src/pages/Faq.js

```diff
@@ -14,78 +14,155 @@ const Faq = () => {
       <QuestionsContainer>
         <h1 className="text-primary display-6">Frequently Asked Questions</h1>
 
-        <h2 className="text-primary display-5">About FightPandemics / General</h2>
+        <h2 className="text-primary display-5">
+          About FightPandemics / General
+        </h2>
         <AnswerContainer>
           <details>
             <summary>What is FightPandemics?</summary>
-            <p>FightPandemics is a platform created amid the COVID-19 outbreak with the aim to be a go-to place for people affected in many ways by the pandemic.</p>
-            <p>Our main mission is to connect individuals and organisations offering and requesting help amid pandemic crisis, in a local and global level. No matter if you seek medical help, facial masks, legal help, groceries, someone to walk your dog or support with any other day-to-day activities, you can look for it here [LINK TO HELP BOARD].</p>
-            <p>If you are sick and think you may have coronavirus disease, you can also find relevant local information [LINK] and check your symptoms [LINK].</p>
-            <p>We strive to gather essential information that is highly fragmented, connecting in one network individuals, communities and organisations working on pandemic-related issues or projects. Our vision is to create a world where everyone is more resilient to emergencies, by helping communities respond more quickly and effectively to health emergencies and humanitarian crises.</p>
+            <p>
+              FightPandemics is a platform created amid the COVID-19 outbreak
+              with the aim to be a go-to place for people affected in many ways
+              by the pandemic.
+            </p>
+            <p>
+              Our main mission is to connect individuals and organisations
+              offering and requesting help amid pandemic crisis, in a local and
+              global level. No matter if you seek medical help, facial masks,
+              legal help, groceries, someone to walk your dog or support with
+              any other day-to-day activities, you can look for it here:
+              <FaqLink href="https://fightpandemics.com/feed" target="_blank">
```

Comment:
didn't you get an ESLint warning that this is required for external links:
`rel="noopener noreferrer"`
Just wondering, I added it to some other link

Reply:
Yeah, I thought so too, but no, for some reason it didn't tell any warnings.